components/crowdfunding-redesign/Profile.tsx

```diff
@@ -133,17 +137,64 @@ export default function Profile({ account }) {
         </div>
       </div>
 
-      <div className="z-10 border-b">
-        <div className="relative mx-auto -mb-px h-16 max-w-screen-xl">
-          <Tabs tabs={['Home', 'Expenses', 'About']} centered={true} />
+      <Tabs defaultValue="home">
+        <div className="border-b" ref={tabRef}>
+          <div className="relative mx-auto -mb-px h-16 max-w-screen-xl px-6">
+            <TabsList centered={true}>
+              <TabsTrigger value="home">Home</TabsTrigger>
+
+              <TabsTrigger value="expenses" count={account.expenses?.totalCount}>
+                Expenses
+              </TabsTrigger>
+              <TabsTrigger value="About">About</TabsTrigger>
+            </TabsList>
+          </div>
         </div>
-      </div>
-      <div className="relative mx-auto max-w-[512px] space-y-4 px-6 py-12">
-        <FundraiserCard account={account} isRoot />
-        {account.childrenAccounts?.nodes?.map(child => {
-          return <FundraiserCard key={child.id} account={child} />;
-        })}
-      </div>
+        <TabsContent value="home">
+          <div className="relative mx-auto max-w-[512px] space-y-4 px-6 py-12">
+            <FundraiserCard account={account} isRoot />
+            {account.childrenAccounts?.nodes?.map(child => {
+              return <FundraiserCard key={child.id} account={child} />;
+            })}
+          </div>
+        </TabsContent>
+        <TabsContent value="expenses">
+          <ExpensesList account={account} />
+        </TabsContent>
+        <TabsContent value="About">
+          <div className="prose prose-slate col-span-7 mx-auto py-12">
+            <Markup
+              noWrap
+              content={profile.longDescription ?? ''}
+              allowAttributes
+              transform={node => {
+                // Allow some iframes
+                const attrs = [].slice.call(node.attributes);
+                if (node.tagName === 'iframe') {
+                  const src = node.getAttribute('src');
+                  const parsedUrl = new URL(src);
+                  const hostname = parsedUrl.hostname;
+                  if (['youtube-nocookie.com', 'www.youtube-nocookie.com', 'anchor.fm'].includes(hostname)) {
+                    const attributes = merge({}, ...attrs.map(({ name, value }) => ({ [name]: value })));
+                    return (
+                      <iframe
+                        {...pick(attributes, ['width', 'height', 'frameborder', 'allowfullscreen'])}
+                        title={attributes.title || 'Embed content'}
+                        src={src}
+                      />
+                    );
+                  }
+                } else if (node.tagName === 'a') {
+                  // Open links in new tab
+                  node.setAttribute('target', '_blank');
+                  node.setAttribute('rel', 'noopener noreferrer');
+                }
```

Comment:
It would be great to have a common helper to share this code with `components/HTMLContent.js`

Reply:
Adding as a follow up